BLSQ/iaso
# IA-2759: Reduce the number of org unit groups

Finally, in my opinion, there was nothing to do in the setuper script except some sql query to clean all groups and recreate only 4 needed in the gpkg file!

Related JIRA tickets : [IA-2759](https://bluesquare.atlassian.net/browse/IA-2759)

## Self proofreading checklist



## Changes

Reduced the number of org unit group in the sample gpkg file


## How to test
From the root folder, open sqlite session: 
   `sqlite3`

Open the database with:

   `.open /setuper/data/small_sample.gpkg`

Check the existing groups with:
     `SELECT * FROM groups;`
  
 Delete all groups with:
       `DELETE FROM groups;`
   
 Create 4 needed groups with:
     `insert into groups (ref, name) values('JM7jsziOLB2', 'Medical Centres/Centres Médicaux'),('bO815RlSIN3','Private hospitals/Cliniques privées'),('IzVj0UdnSRF','Nursing facility/Soins infirmiers'),('FnGKoaZpj4u','Public hospitals/Hôpitaux publics');`
   
Close sqlite session with `ctrl+z`

Launch the process defined in the following link to create test data with command  `python setuper` : https://github.com/BLSQ/iaso/blob/main/setuper/README.md

You should see only 4 created groups from the group page: http://localhost:8081/dashboard/orgunits/groups/accountId/1/order/name/pageSize/20/page/1

## Print screen / video

Once the test data created, check the groups list from the groups page :

https://github.com/BLSQ/iaso/assets/6383219/a1389bee-7c52-48d3-907f-bc7d09b7f3e1



[IA-2759]: https://bluesquare.atlassian.net/browse/IA-2759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
